DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -68,7 +68,7 @@
 
 
   <class name="pat::Muon"  ClassVersion="29">
-   <version ClassVersion="29" checksum="4221614933"/>
+   <version ClassVersion="29" checksum="3276039509"/>
```

Comment:
I think you need to add a new class version with the new checksum, rather than modifying version 29